### ISSUE TITLE
Bootstrapping ?

### ISSUE THREAD

**Issue by astik** [io-informatics/angular-jsonld#1]
Hi,
Is there any chance you can provide some documentation for how to bootstrap an Angular application that would consume JSON-LD API ?
Thanks in advance =)

**Comment by glibersat** [comment]
:+1: on that! I've managed to get a LDPR from my LDP Server but so far I can't see the real benefit of using angular-jsonld... reading the code, I'm sure there are interesting features but it takes time to decypher the code. Just a simple example of each feature would be awesome. Thanks :)

**Comment by astik** [comment]
Indeed, I try to dive into the source code but lacking of time right now I couldn't go far ='(
Also, I'm no expert in JSON-LD and I would love to undeerstand why this kind of layer is necessary. I heard that is necessary but I don't why and I don't like not to understand =D

**Comment by alexdeleon** [comment]
Hi guys,
Thanks bringing up this issue. Yes we urgently need a "get started" documentation! I will get it up as soon as possible. Give a day or two. I ping you here when its up.

**Comment by glibersat** [comment]
Awesome, thanks @alexdeleon !

**Comment by alexdeleon** [comment]
Hi,
I committed a basic README file which intends to describe the two main _reasons_ for this project to exist. I am really interested in your opinion about these features. Is a library like this needed on the client side? Is it worth the effort? I should also mentioned that I started this project because I wanted to use [Hydra](http://www.hydra-cg.com) to build my APIs ([see this related article](http://www.markus-lanthaler.com/research/hydra-a-vocabulary-for-hypermedia-driven-web-apis.pdf)) . I will document more about using hydra with this library later.

Looking forward for your comments.

Cheers!
Alex
